tap_spreadsheets_anywhere/csv_handler.py

```diff
@@ -13,8 +13,8 @@ def generator_wrapper(reader):
 
             formatted_key = key
 
-            # remove non-word, non-whitespace characters
-            formatted_key = re.sub(r"[^\w\s]", '', formatted_key)
+            # replace non-word, non-whitespace characters with underscores
+            formatted_key = re.sub(r"[^\w\s]", '_', formatted_key)
```

Comment:
Let's have a `+` at the end of the regexes, so that multi-character non-word/non-space strings get replaced by a single underscore.

Reply:
Actually, instead, how about combining the two substitutions into one, using `r"\W+"` as the pattern.